src/app/_pages/housing-availables/housing-availables.page.ts

```diff
@@ -0,0 +1,53 @@
+import { Component, ViewChild } from '@angular/core';
+import { ApiService, HeadService } from '_services/index';
+import { HousingAvailable } from '_models/housing-available.model';
+import { Body } from '@angular/http/src/body';
```

Comment:
What is this for? I don't _think_ it's needed, but you can explain if it is.

Reply:
This declares an empty array of type HousingAvailable, a type established and exported by '_models/housing-available.model'.  In our meeting you had instructed me to include this, but told me to type it as 'any', which i found didn't work; however, the moment I switched the array to have a declared type of HousingAvailable[], Typescript liked it and it ran fine for the first time (displaying the cards and their info).